packages/clay-color-picker/src/index.tsx

```diff
@@ -88,6 +93,11 @@ interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
 	 */
 	onColorsChange?: (val: Array<string>) => void;
 
+	/**
+	 * Callback for when the validity of the color value changes
+	 */
+	onValidation?: (val: boolean) => void;
+
```

Comment:
You probably forgot to remove 😅

Reply:
yep! I rely on eslint too much sometimes lol